packages/ember-htmlbars/tests/helpers/if_unless_test.js

```diff
@@ -51,248 +47,6 @@ QUnit.module('ember-htmlbars: {{#if}} and {{#unless}} helpers', {
   }
 });
 
-QUnit.test('The `if` helper tests for `isTruthy` if available', function() {
```

Comment:
Replaced by:

* "it tests for `isTruthy` if available" 
* "it tests for `isTruthy` on Ember objects if available"

(Now testing re-render, mutation and replacement as well)